Modules/Challenges/test/Challenges.API.Tests.Performance.EndToEnd/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "typescript",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "ssh://git@github.com/k6io/example-typescript.git",
+  "author": "Simon Aronsson <simme@k6.io>",
+  "license": "MIT",
+  "devDependencies": {
+    "@babel/core": "7.13.16",
+    "@babel/plugin-proposal-class-properties": "7.13.0",
+    "@babel/plugin-proposal-object-rest-spread": "7.13.8",
+    "@babel/preset-env": "7.13.15",
+    "@babel/preset-typescript": "7.13.0",
+    "@types/k6": "~0.41.0",
+    "@types/webpack": "5.28.0",
+    "babel-loader": "8.2.2",
+    "clean-webpack-plugin": "4.0.0-alpha.0",
+    "copy-webpack-plugin": "^9.0.1",
+    "k6-expect": "^0.0.15",
+    "typescript": "4.2.4",
+    "webpack": "^5.75.0",
+    "webpack-cli": "4.6.0",
+    "webpack-glob-entries": "^1.0.1"
+  },
+  "scripts": {
+    "get-test": "k6 run -e HOST=http://localhost:5000 -e VUS=1 -e ITERATIONS=100 -e CLIENT_SECRET=test dist/get.test.js",
+    "post-with-authentication": "k6 run -e HOST=http://localhost:5000 -e VUS=1 -e ITERATIONS=100 -e CLIENT_SECRET=test dist/post.with.authentication.test.js",
+    "post-without-authentication": "k6 run -e HOST=http://localhost:5000 -e VUS=1 -e ITERATIONS=100 dist/post.without.authentication.test.js",
```

Comment:
The three scripts that run the individual scripts don't execute webpack, which could cause unwanted behavior, when you change the tests and run the scripts without building manually first.

The problem is: if you just add `webpack &&`, then `webpack` is executed 4 times in total if you run the `start` script. But if you manage to change the `start` script in the way that I suggested in the other comment (run all tests instead of run the specified ones), then this isn't a problem anymore.